pr_check.sh

```diff
@@ -10,7 +10,7 @@ export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build'
 export IMAGE_TAG=$(git rev-parse --short=7 HEAD)
 export GIT_COMMIT=$(git rev-parse HEAD)
 export APP_ROOT=$(pwd)
-export NODE_BUILD_VERSION=15
+export NODE_BUILD_VERSION=18
```

Comment:
We should also bump the version for travis while we are at it. Do watch out bumping node on travis will also require bumping the distro. Take a look at the compliance travis config.

Reply:
theres a difference in script sections and the dist key/value but overall the only other difference is the string/int in node version but travis is smart enough to read that. What else would you think should mimic compliance? currently inventory travis has a lot of if statements in there. We could remove that overall

Reply:
... I am not sure what you mean... :D These three lines from the compliance travis config should be in the `.travis.yml` here to get node 18: https://github.com/RedHatInsights/compliance-frontend/blob/master/.travis.yml#L5-L7

Reply:
Seems my eyes were broken, added lol